homeassistant/components/remote_calendar/strings.json

```diff
@@ -18,19 +18,19 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
     },
     "error": {
-      "unsupported_protocol": "Request URL is missing an 'http://' or 'https://' protocol",
-      "url_not_reachable": "URL is not reachable",
-      "not_authorized": "You are not authorized to access this URL",
+      "unsupported_protocol": "Request URL is missing an 'http://' or 'https://' protocol.",
+      "url_not_reachable": "URL is not reachable.",
```

Comment:
From a user perspective, i think this distinction is not needed and can use `cannot_connect`